app/views/petitions/_rejected_petition.html.erb

```diff
@@ -0,0 +1,21 @@
+<h1>
+  <span class="heading-secondary">Rejected petition</span>
+  <%= @petition.title %>
```

Comment:
Providing a partial local param gets my vote.  Relying on globals is a bad idea.